openl3/cli.py

```diff
@@ -124,21 +171,29 @@ def parse_args(args):
     parser.add_argument('--input-repr', '-i', default='mel256',
                         choices=['linear', 'mel128', 'mel256'],
                         help='String specifying the time-frequency input '
-                             'representation for the embedding model.')
+                             'representation for the audio embedding model.')
 
     parser.add_argument('--content-type', '-c', default='music',
                         choices=['music', 'env'],
                         help='Content type used to train embedding model.')
 
-    parser.add_argument('--embedding-size', '-s', type=int, default=6144,
-                        help='Embedding dimensionality.')
+    parser.add_argument('--audio-embedding-size', '-as', type=int, default=6144,
+                        choices=[6144, 512],
+                        help='Audio embedding dimensionality.')
 
-    parser.add_argument('--no-centering', '-n', action='store_true', default=False,
-                        help='Do not pad signal; timestamps will correspond to '
+    parser.add_argument('--no-audio-centering', '-n', action='store_true',
+                        default=False,
+                        help='Used for audio embeddings. Do not pad signal; '
+                             'timestamps will correspond to '
                              'the beginning of each analysis window.')
```

Comment:
Well currently there is no alignment, we simply compute the embeddings separately: 1) an image embedding for every frame of the video and 2) audio embeddings produced at the the given hop size. We could have an align mode, though this would be an entirely new feature to develop and I'd prefer to put this off to another minor release.

Reply:
OK, thoughts:
* I think an "align" mode would be super helpful, since people processing the video (audio + image) are probably looking at multimodal processing
* I agree it's a new feature though, so can be kept for a future release
* However, for this release we should make it super clear in the docs that the extracted embeddings are NOT aligned, and always return timestamps for each modality

Reply:
:+1:

Reply:
I've deferred this to #32 since that is the PR in which I actually update the tutorial, but in the mean time I've updated the docstring to `process_video_file` to clarify this.